src/spec-node/featuresCLI/test.ts

```diff
@@ -10,7 +10,8 @@ import { doFeaturesTestCommand } from './testCommandImpl';
 export function featuresTestOptions(y: Argv) {
 	return y
 		.options({
-			'features': { type: 'array', alias: 'f', describe: 'Feature(s) to test as space-separated parameters. Omit to run all tests.  Cannot be combined with \'--global-scenarios-only\'.', },
+			'target-project': { type: 'string', alias: 't', default: '.', description: 'Path to folder containing \'src\' and \'test\' sub-folders.' },
```

Comment:
`--workspace-folder` or `--project-folder` might make it easier to read since it is more obvious that it is about a folder (and not a name or some other kind of reference).

Reply:
I didn't want to conflate with the concept of `workspace-folder`, but I agree that `project-folder` is more clear - will change!